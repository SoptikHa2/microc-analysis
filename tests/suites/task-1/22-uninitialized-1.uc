 main() {
   var x;
   return x;
 }
