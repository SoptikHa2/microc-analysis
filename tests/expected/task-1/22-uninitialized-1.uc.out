---
test: run suites/task-1/22-uninitialized-1.uc
---
Analysis found some problems. Aborting run.
UninitIdentifier "In main, \"suites/task-1/22-uninitialized-1.uc\" (line 3, column 10): Accessing local x that is not initialized."

exit: 1
